adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -84,11 +84,19 @@ class TransformArgs extends Args4jBase with ADAMSaveAnyArgs with ParquetArgs {
   var forceLoadParquet: Boolean = false
   @Args4jOption(required = false, name = "-single", usage = "Saves OUTPUT as single file")
   var asSingleFile: Boolean = false
+  @Args4jOption(required = false, name = "-paired_fastq", usage = "When converting two (paired) FASTQ files to ADAM, pass the path to the second file here.")
+  var pairedFastqFile: String = null
+  @Args4jOption(required = false, name = "-record_group", usage = "Set converted FASTQs' record-group names to this value; if empty-string is passed, use the basename of the input file, minus the extension.")
+  var fastqRecordGroup: String = null
+  @Args4jOption(required = false, name = "-concat", usage = "Concatenate this file with <INPUT> and write the result to <OUTPUT>")
+  var concatFilename: String = null
```

Comment:
well, in the paired-fastq case (cf. [the example cmdline on the main thread](https://github.com/bigdatagenomics/adam/pull/831#issuecomment-144555308)) I don't think using a glob as the `<INPUT>` argument helps me: in the presence of the `-paired_fastq` flag, the `<INPUT>` is interpreted as a FASTQ with the first-of-pair reads; the `-concat` flag is used to add some unpaired reads in to the same `.adam` as the paired reads.

The glob syntax would result in trying to `zip` up one `RDD` that is [first-of-pair and unpaired reads] with a second that is [second-of-pair] reads.

In any case, allowing for an explicit way to merge two files together in `transform`, which the `-concat` flag does here, seems worth doing, e.g. for cases where a glob is too coarse a mechanism.

I do have some desire to make an `ADAMCommand` that will merge two *or more* `RDD[AlignmentRecord]` files, since in the 200x Illumina Platinum data there are many FASTQs from each individual and I want to merge them into per-individual FASTQs, but I'd open a separate issue/PR for that.